GSA/digitalgov.gov
# Youtube video embed update

## Summary

TL;DR -> removed autoplay from youtube embed url and update styles to match Digital theme 

Removed the `?autoplay=1` from the youtube embedded urls to help address the videos playing once a user loads the page.  Also updated the background color or the video to match one of Digital color themes.  

## Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-video-embeded-update//event/2024/06/20/uswds-monthly-call-june-2024/)



## Solution

Added the BEM naming of `dg-youtube` and removed the uswds class of `stage`, also moved the styles from stage class to the new `dg-youtube` class for better control of styles in the future.

Also removed the autoplay within the url to address an issue with the video playing once a user loaded the page




## How To Test

1. visit [Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-video-embeded-update//event/2024/06/20/uswds-monthly-call-june-2024/) and check that the video is not autoplaying, also verify that the video plays with now error 
2. inspect element and verify that the new `dg-youtube` class is applied and the new background color is `#ecf1f7`
